iac-extensions/spring-config/src/main/antlr/PropertiesParser.g4

```diff
@@ -45,6 +46,11 @@ eol
     | EOF
     ;
 
+// The lines that starts from delimiter doesn't make much sense but Spring accept such entries without any error
```

Comment:
I suggest to also mention that this is a separate parser rule because we don't want to insert such lines into the AST in the visitor from the regular `visitLine` method (at least this is how I understand it)